### PR TITLE
[QA] Fix variable refs in HOMEPAGE

### PR DESCRIPTION
**Pull request by mgorny**
CC @gentoo/qa 

Note that it doesn't fix all ebuilds, just the low-hanging fruit for now.

### CODE REVIEW

**Comment by flappyports** [comment]
@mgorny actually, it seems you were only targeting ${PN}. There are a few using ${MY_PN}.

File: dev-java/objenesis/objenesis-1.3.ebuild [hunk 1]
```diff
@@ -8,7 +8,7 @@ JAVA_PKG_IUSE="source doc"
 inherit java-pkg-2 java-pkg-simple
 
 DESCRIPTION="A small Java library with one purpose: To instantiate a new object of a class"
-HOMEPAGE="https://${PN}.googlecode.com/"
+HOMEPAGE="https://objenesis.googlecode.com/"
```
Comment (SoapGentoo) [inline dev-java/objenesis/objenesis-1.3.ebuild]: fine, still useless though

File: sci-mathematics/pymc/pymc-2.3.6.ebuild [hunk 2]
```diff
@@ -8,7 +8,8 @@ PYTHON_COMPAT=( python{2_7,3_5,3_6} )
 inherit distutils-r1 flag-o-matic fortran-2 toolchain-funcs
 
 DESCRIPTION="Markov Chain Monte Carlo sampling toolkit"
-HOMEPAGE="https://github.com/${PN}-devs/${PN} https://pypi.org/project/${PN}"
+HOMEPAGE="https://github.com/pymc-devs/pymc
```
Comment (SoapGentoo) [inline sci-mathematics/pymc/pymc-2.3.6.ebuild]: this ebuild is going anyways

File: dev-java/objenesis/objenesis-1.3.ebuild (hunk 1 above)
Comment (flappyports) [reply dev-java/objenesis/objenesis-1.3.ebuild]: I think the point was to make CI happy now

**Comment by mgorny** [comment]
It's just a trivial sed. It's easier to modify everything, and I fine it a waste of time to revert changes on things lastrited or broken.

**Pull request closed without merging**